pkg/liquidity-mining/contracts/gauges/ChildChainGauge.vy

```diff
@@ -115,8 +115,9 @@ inflation_rate: public(HashMap[uint256, uint256])
 
 
 @external
-def __init__(_bal_token: address, _factory: address, _authorizer_adaptor: address):
+def __init__(_bal_token: address, _factory: address, _authorizer_adaptor: address, _version: String[128]):
     self.lp_token = 0x000000000000000000000000000000000000dEaD
+    self.version = _version
 
     BAL = _bal_token
     FACTORY = _factory
```

Comment:
Shouldn't we remove this?

Reply:
This will be addressed in #2306, yes.
We'll be recycling the factory reference in favor of the BAL pseudo minter.